### ISSUE TITLE
Reveal password icon is the wrong color in dark mode

### ISSUE THREAD

**Issue by styris-ame** [amwx/FluentAvalonia#523]
**Describe the bug**
Reveal password button icon is the wrong color when using dark mode. Note that on hover it turns to the correct color (white) and back to the wrong color (black) when unhovered.

**Screenshots**
![FAControlsGallery Desktop_dagxcqQKrW](https://github.com/amwx/FluentAvalonia/assets/123859041/83a7bbf8-2451-4b6e-90dc-18f81ded4026)

**Desktop/Platform (please complete the following information):**
 - OS: Windows 11
 - FluentAvalonia Version Latest (cloned)
 - Avalonia Version Latest (cloned)

**Comment by robloo** [comment]
Note that this issue also occurs in some other places. For example, the NavigationView.PaneCustomContent (with a button).

![image](https://github.com/amwx/FluentAvalonia/assets/17993847/00623eae-8d8a-4756-a5bf-ca76e387caa2)

**Comment by amwx** [comment]
Fixed in the above commit

@robloo I'm not seeing an issue testing this myself, and there are no setters that force a foreground around the PaneCustomContent. The TextBox button issue was a result of me originally hardcoding the foreground for the icon within the template, I've moved that out to setters and dynamic resources. Have you checked your button styles to make sure you're not accidentally using static resource or something that causes the button icon to render in black text?
